x/pki/handler_test.go

```diff
@@ -412,11 +412,20 @@ func TestHandler_ApproveAddX509RootCert_ForNotEnoughApprovals(t *testing.T) {
 
 	// query approved certificate and we should get one back
```

Comment:
Do we have a unit test where there are multiple certificate with the same SubjectKeyId (but different subjects) are added? We need to check that they all are correctly returned (not overridden).
Also, we need to check that if a certificate with subject1-subject-key-id2 is revoked, subject2-subject-key-id is not removed and can still be queried by subject-key-id.

Reply:
Please look at `x/pki/handler_test.go:663`, `x/pki/handler_test.go:1513`